hadoop-hdds/common/src/main/proto/SCMSecurityProtocol.proto

```diff
@@ -21,6 +21,7 @@
  * Please see http://wiki.apache.org/hadoop/Compatibility
  * for what changes are allowed for a *unstable* .proto interface.
  */
+syntax = "proto2";
```

Comment:
I didnt' understand it fully, if we are compiling the files using proto3, can we also move the format to proto3 as well ?

Reply:
Compiling only. So no, we cannot move to Protoc3 in this patch. That would require some serious testing and work. But it is an amazing goal to have. I have probably try to change the DN protocol since it is on GRPC and independent of Hadoop RPC libraries. Eventually, we will get there, but baby steps.

Reply:
Agree. proto3 syntax is different. Easier to upgrade the compilation library first and update the proto files to syntax 3 in the next step.

Reply:
Thanks makes sense.